tutorials/nservicebus-101/lesson-2/tutorial.md

```diff
@@ -0,0 +1,182 @@
+---
+title: "NServiceBus 101 Lesson 2: Sending a command"
+---
+
+Sending and receiving messages is at the core of any NServiceBus system. Durable messages passed between processes allow reliable communication between those processes, even if one of them is temporarily unavailable. In this lesson we'll show how to send and process a message.
+
+
+## Objectives
+
+By the end of this lesson, you will have learned how to:
+
+ * Define messages
+ * Define message handlers
+ * Send and receive a message locally
+ * Use NServiceBus's logging capabilities
+
+
+## What is a message?
+
+A [**message**](/nservicebus/messaging/messages-events-commands.md) is a collection of data sent via one-way communication between two endpoints. In NServiceBus, we can define a message via a simple class.
```

Comment:
> is a collection of data ... we can define a message via a simple class...

I'd stress what Szymon pointed out earlier - that message is a data contract, **should be** (not just could) simple POCO class, etc.